### ISSUE TITLE
eselect-opengl-1.3.3_p1 "Invalid symbolic links"

### ISSUE THREAD

**Issue by EoD** [sjnewbury/gentoo-gpu#14]
```
$ eselect opengl list
!!! Error: Invalid symbolic links: OpenGL implementation "" symbolic links are missing.
Available OpenGL implementations:
  (none found)
exiting
```

Any more information I can provide?

**Comment by sjnewbury** [comment]
Hopefully fixed now with changes to x11-drivers/nvidia-drivers.  Please report back if still having issues, Sorry for taking so long, been busy playing Elite Dangerous! ;-)

**Comment by EoD** [comment]
@sjnewbury no sorry. I never had the nvidia-drivers installed and with ``=app-eselect/eselect-opengl-1.3.3_p1::gentoo-gpu`` I am still getting
```
# eselect opengl list
!!! Error: Invalid symbolic links: OpenGL implementation "" symbolic links are missing.
Available OpenGL implementations:
  (none found)
exiting
```

while ``app-eselect/eselect-opengl-1.3.1-r4::gentoo`` yields
```
# eselect opengl list
Available OpenGL implementations:
  [1]   xorg-x11
```

**Comment by sjnewbury** [comment]
Weird.

I get:

eselect opengl list
Available OpenGL implementations:
  [1]   glvnd
  [2]   nvidia
  [3]   primus
  [4]   xorg-x11 *

**Comment by sjnewbury** [comment]
Did you install mesa from the gentoo-gpu overlay too?

**Comment by EoD** [comment]
@sjnewbury No, I am running a (more or less) stable system and ``media-libs/mesa-9999-r1::gentoo-gpu`` from gpu-overlay is masked, while ``app-eselect/eselect-opengl-1.3.3_p1::gentoo-gpu`` is stable.

**Comment by EoD** [comment]
@sjnewbury I installed ``media-libs/mesa-9999-r1::gentoo-gpu`` and I am experiencing the same error.

**Comment by sjnewbury** [comment]
I need to mark eselect-opengl-1.3.3_p1 as unstable or un-KEYWORD it entirely.  It works well for me, but it shouldn't get pulled in automatically for people running "stable".

**Comment by sjnewbury** [comment]
I'll see if I can help work out why it's not working though, but I can't do that until later this afternoon.

**Comment by sjnewbury** [comment]
@EoD Can you run "eselect --debug opencl list" and attach the output to this bug?

**Comment by EoD** [comment]
@sjnewbury not sure what you are up to here
```
$ eselect --debug opengl list
!!! Error: Unknown option --debug
exiting
```

**Comment by sjnewbury** [comment]
@EoD Hmm works for me... okay I've got debug available because I'm using the enhanced eselect from the "science" overlay.  Can you install that and see?

**Comment by EoD** [comment]
I don't like the idea of installing another overlay in order to run gentoo-gpu. 

I tried it nevertheless and I got the following conflicts, which I do not want to remove now:
```
Calculating dependencies... done!
[ebuild     U ~] app-admin/eselect-1.4.10-r100::science [1.4.8::gentoo] USE="vim-syntax -doc -emacs" 174 KiB
[blocks B      ] app-eselect/eselect-lapack ("app-eselect/eselect-lapack" is blocking app-admin/eselect-1.4.10-r100)
[blocks B      ] app-eselect/eselect-blas ("app-eselect/eselect-blas" is blocking app-admin/eselect-1.4.10-r100)
```

Would be great if you could mask your package first and then try to fix this bug here (as it seems to be more complicated).

**Comment by sjnewbury** [comment]
It should be keyword masked already, I changed it yesterday.

That's fair enough, I don't want you to break your system!  I was just hoping for something more to go on since I'd like to know why it isn't working for you.
